### PR TITLE
duration after changing video quality

### PR DESCRIPTION
**Pull request by melbahja**
continue at the current duration after changing video quality

### Checklist
- [x] Use `develop` as the base branch
- [x] Exclude the gulp build (`/dist` changes) from the PR
- [x] Test on: chrome, firefox

### CODE REVIEW

File: src/js/plyr.js [hunk 1]
```diff
@@ -680,6 +680,8 @@ class Plyr {
     set quality(input) {
         const config = this.config.quality;
         const options = this.options.quality;
+        const duration = this.duration;
+        const isPlaying = this.playing;
```
Comment (sampotts) [inline src/js/plyr.js]: Can we not use object destructuring here? e.g. 

`const { duration, playing } = this;`

File: src/js/plyr.js [hunk 2]
```diff
@@ -703,6 +705,14 @@ class Plyr {
 
         // Set quality
         this.media.quality = quality;
+        
+        // seek to duration before changing quality
```
Comment (sampotts) [inline src/js/plyr.js]: It'd be good to start comments with a capital to match the others

**Comment by sampotts** [comment]
Thanks for this 👍

**Comment by sampotts** [comment]
Remind me why we want to seek to the end of the video on source change? I've backed this one out as I don't think there's any benefit to it. What was the issue you were trying to solve?

**Comment by melbahja** [comment]
@sampotts when you change the video quality plyr starts from the beginning
This PR make the video continue from the point before changing the quality

**Comment by sampotts** [comment]
I can't reproduce that. Anyway `duration` is the _length_ of the media, I think you meant `currentTime` but that's already handled. For me it seeks back to where it was previously.

**Comment by melbahja** [comment]
yes its `currentTime` before v3.4.5 there was an issue when change the quality. now i saw plyr.io demo it's working fine

**Comment by sampotts** [comment]
OK. Yeah, your code was using `duration` instead of `currentTime` ...

```javascript
// Seek to duration before changing quality
this.seek = duration;
```

Anyway, glad it's sorted. Nothing has changed with that recently. Perhaps you were using an old version.

**Pull request merged**